azkaban-common/src/main/java/azkaban/project/ProjectManager.java

```diff
@@ -268,16 +232,10 @@ public Project createProject(String projectName, String description,
           "Project names must start with a letter, followed by any number of letters, digits, '-' or '_'.");
     }
 
-    if (projectsByName.containsKey(projectName)) {
```

Comment:
azkaban.project.JdbcProjectLoader#createNewProject(java.sql.Connection, java.lang.String, java.lang.String, azkaban.user.User) will throw the same exception.